source/includes/aws/_nodegroups.md

```diff
@@ -0,0 +1,118 @@
+      "desiredNodeCount": 1
+    },
+    {
+      "id": "cluster-2/nodegroup-1",
+      "clusterName": "cluster-2",
+      "name": "nodegroup-1",
+      "status": "ACTIVE",
+      "version": "1.22",
+      "capacityType": "ON_DEMAND",
+      "amiType": "AL2_x86_64",
+      "instanceTypes": [
+        "t2.nano"
+      ],
+      "desiredNodeCount": 1
+    }
+  ],
+  "metadata": {
+    "recordCount": 3
+  }
+}
+```
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/nodegroups</code>
+
+Attributes | &nbsp;
+------- | -----------
+`id` <br/> *string* | The nodegroup is uniquely identified by its name.
+`clusterName` <br/> *string* | The name of the cluster that the nodegroup belongs to.
+`name` <br/> *string* | The name of the nodegroup.
+`status` <br/> *string* | The status of the nodegroup.
+`version` <br/> *string* | The version of Kubernetes that the nodegroup is using.
+`capacityType` <br/> *string* | The capacity type that the nodegroup will use, can either support on-demand, or spot instances.
+`amiType` <br/> *string* | The Amazon Machine Image (AMI) type that the nodegroup will use.
+`instanceTypes` <br/> *List&lt;string&gt;* | The list of instance types that this nodegroup's generated nodes can have
+`desiredNodeCount` <br/> *integer* | The total amount of virtual CPUs across all nodes in the cluster.
+
+<!-------------------- RETRIEVE A NODEGROUP -------------------->
+
+#### Retrieve a nodegroup
+
+```shell
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/api/v2/services/aws/test-area/nodegroups/cluster-1/nodegroup-1"
+```
+> The above command returns a JSON structured like this:
+
+```json
+{
+  "data": {
+    "id": "cluster-1/nodegroup-1",
+    "clusterName": "cluster-1",
+    "name": "nodegroup-1",
+    "status": "ACTIVE",
+    "version": "1.22",
+    "capacityType": "ON_DEMAND",
+    "amiType": "AL2_x86_64",
+    "instanceTypes": [
+      "t2.nano"
+    ],
+    "desiredNodeCount": 1
+  }
+}
+```
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/nodegroups/nodegroupId</a></code>
+
+
+Attributes | &nbsp;
+------- | -----------
+`id` <br/> *string* | The nodegroup is uniquely identified by its name.
+`clusterName` <br/> *string* | The name of the cluster that the nodegroup belongs to.
+`name` <br/> *string* | The name of the nodegroup.
+`status` <br/> *string* | The status of the nodegroup.
+`version` <br/> *string* | The version of Kubernetes that the nodegroup is using.
+`capacityType` <br/> *string* | The capacity type that the nodegroup will use, can either support on-demand, or spot instances.
+`amiType` <br/> *string* | The Amazon Machine Image (AMI) type that the nodegroup will use.
+`instanceTypes` <br/> *List&lt;string&gt;* | The list of instance types that this nodegroup's generated nodes can have
```

Comment:
Missing period at then end of the instanceTypes definition

Reply:
Fixed 👌🏻